packages/opc-header/src/opc-header.scss

```diff
@@ -0,0 +1,36 @@
+//@import "../../opc-styles/opc-styles.scss";
```

Comment:
```suggestion
@import "../../opc-styles/opc-styles.scss";
```

Reply:
I think a better option would be to use an npm package for `opc-styles.scss`.\
That will make it easier to publish `opc-header` as a standalone component.

Reply:
lit-elements has a known issue with @import: https://github.com/Polymer/lit-element/issues/559

Reply:
@sayak-sarkar @deshmukhmayur This issue has now been resolved by using https://www.npmjs.com/package/@one-platform/opc-styles